roles/aks-apply/files/dev/otp-data-builder-finland-v3-prebuilt-graph-dev.yml

```diff
@@ -0,0 +1,108 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: otp-data-builder-finland-v3-prebuilt-graph
```

Comment:
Maybe we should shorten the names so something like `otp-transit-builder-finland-v3` and the other one would be `otp-street-builder-finland-v3`?

Reply:
I changed the names to these